Cargo.toml

```diff
@@ -131,6 +131,12 @@ unnecessary_join = "deny"
 unnecessary_lazy_evaluations = "deny"
 unused_async = "deny"
 
+explicit_deref_methods = "deny"
```

Comment:
Why not keep all lints in the same block, sorted by name?

(while there move the newline from line 113 to above the comment that precedes `single_char_pattern`.)

Reply:
OK, done.